src/frontend/src/index.less

```diff
@@ -1,4 +1,69 @@
 @nodeModulesPath: './../../';
 @coreModulePath: './../../';
 
-@import '../node_modules/nav-frontend-knapper-style/src/index';
\ No newline at end of file
+@import '../node_modules/nav-frontend-knapper-style/src/index';
+@import '../node_modules/nav-frontend-paneler-style/src/index';
+
+@import '../node_modules/nav-frontend-skjema-style/src/index';
+
+@import './start/start';
+
+body {
+    background-color: #e9e7e7;
+}
+
+.body__wrapper {
+    width: 50%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 2rem;
+
+    @import './skjema/skjema';
+}
+
+article.registrerdeg {
+    margin: 5em auto 0 auto;
+    max-width: 50em;
+    color: #59514B;
+
+    > section {
```

Comment:
Style på klassenavn i stedet for på tager. Da vil du ikke brekke stylingen om du endrer litt på htmlen

Reply:
Jeg skal fikse på den nå